### PR TITLE
October release notes

### PR DESCRIPTION
**Pull request by danabauer**
## What's in this PR?
- october release notes
- september release notes with updated slug

### Docs Preview:

[Docs preview for this PR.](https://dfhx9f55j8eg5.cloudfront.net/how-to/pr/122)

View all staged runs:
https://github.com/OvertureMaps/docs/actions/workflows/publish-pr-to-staging.yml

### CODE REVIEW

File: release-blog/2024/2024-10-23.0.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+- fixed building height but there is no source information 
+- fixed building heights derived from lidar in areas with tree cover are incorrect 
+- fixed ML building in water 
+- fixed lidar heights incorrectly being added 
+- added new data from OSM (2024-10-05)
+- added new data from Microsoft (2024-09-24)
+- added new from Esri (2024-10-15)
+
+### Divisions
+- changed subtype of Puerto Rico to dependency
```
Comment (stepps00) [inline release-blog/2024/2024-10-23.0.mdx:87]: This should change to "updated the subtype of 38 dependencies" or similar. The theme maintained these places as countries, but in this release, they're (more correctly) assigned a dependency subtype.

File: release-blog/2024/2024-10-23.0.mdx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+:::info
+The base, buildings, divisions, and places themes are in GA. The transportation theme is in beta, and the addresses theme is in alpha.
+:::
+
+### Addresses
+- added new address features in the following countries:
+    - **89.9 million** in Brazil
+    - **1.7 million** in Croatia
+    - **5.9 million** in German[y](https://youtu.be/WBpDs-BfwO0?feature=shared)
```
Comment (stepps00) [inline release-blog/2024/2024-10-23.0.mdx:61]: `**5.9 million** in Germany`

File: release-blog/2024/2024-10-23.0.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- changed subtype of Puerto Rico to dependency
+- fixed microhood/neighborhood discrepancy between feature types
+- updated regions in North Macedonia
+- moved features with a `suburbs` tag under a new `macrohood` subtype
+
+### Places
+- no updates
+
+### Transportation
+- removed `connector_ids` property; more information [here](https://docs.overturemaps.org/blog/2024/08/15/preview-august-release/#connectors).
```
Comment (stepps00) [inline release-blog/2024/2024-10-23.0.mdx:96]: remove period for consistency?

File: release-blog/2024/2024-10-23.0.mdx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+- added data from the [NYC Open Data](https://opendata.cityofnewyork.us/) portal
+- fixed a bug that resulted in both the US National Address Database and NYC Open Data being included for Queens
+- fixed a typo in the Geoscape Australia source name
+- changed the data source name for Chile to CL-INE and Portugal to PT-INE to distinguish the two
+
+### Base
+- fixed lake features in Manitoba, Canada
+
+### Buildings
+- fixed building features with class but no subtype · Issue #183 · OvertureMaps/data
```
Comment (stepps00) [inline release-blog/2024/2024-10-23.0.mdx:76]: fixed building features with class but no subtype ([issue](https://github.com/OvertureMaps/data/issues/183))

**Review by stepps00 — APPROVED** [review]
A few minor nits, but otherwise looking great.

**Review by jenningsanderson — APPROVED** [review]

**Comment by jenningsanderson** [comment]
I added the rest of the m10 updates into this single PR

**Pull request merged**